data/workspace.json

```diff
@@ -1 +1,640 @@
+        "examples": [
+          {
+            "text": "I need to return"
+          },
+          {
+            "text": "I want to return something"
+          },
+          {
+            "text": "I want my money back"
+          },
+          {
+            "text": "I purchased, but"
+          },
+          {
+            "text": "I want to return my shoes"
+          },
+          {
+            "text": "I'd like to return"
+          },
+          {
+            "text": "I don't want"
+          },
+          {
+            "text": "I have bought, but"
+          }
+        ]
+      },
+      {
+        "intent": "Shop",
+        "examples": [
+          {
+            "text": "continue"
+          },
+          {
+            "text": "shop"
+          },
+          {
+            "text": "I want"
+          },
+          {
+            "text": "continue shopping"
+          },
+          {
+            "text": "I'm shopping for"
+          },
+          {
+            "text": "I'm looking for"
+          },
+          {
+            "text": "Do you have"
+          },
+          {
+            "text": "Can I get"
+          }
+        ]
+      },
+      {
+        "intent": "deleteItem",
+        "examples": [
+          {
+            "text": "I would like to delete item"
+          },
+          {
+            "text": "delete item"
+          },
+          {
+            "text": "delete"
+          }
+        ],
+        "description": ""
+      },
+      {
+        "intent": "startOver",
+        "examples": [
+          {
+            "text": "start again"
+          },
+          {
+            "text": "start over"
+          },
+          {
+            "text": "try again"
+          },
+          {
+            "text": "begin again"
+          }
+        ]
+      },
+      {
+        "intent": "checkout",
+        "examples": [
+          {
+            "text": "checkout"
+          },
+          {
+            "text": "payment"
+          },
+          {
+            "text": "complete"
+          }
+        ],
+        "description": ""
+      }
+    ],
+    "entities": [
+      {
+        "entity": "product",
+        "values": [
+          {
+            "type": "synonyms",
+            "value": "shoe",
+            "synonyms": [
+              "boot",
+              "boots",
+              "clogs",
+              "pair of shoes",
+              "pumps",
+              "sandals",
+              "shoes"
+            ]
+          },
+          {
+            "type": "synonyms",
+            "value": "shirt",
+            "synonyms": [
+              "blouse",
+              "button down",
+              "polo",
+              "top"
+            ]
+          },
+          {
+            "type": "synonyms",
+            "value": "pants",
+            "synonyms": [
+              "capris",
+              "slacks"
+            ]
+          }
+        ]
+      },
+      {
+        "entity": "NegativeResponse",
```

Comment:
The bot control stuff could/should come from the content catalog as an best practices example.  There are affirmative/negative responses, start over and things like that in there for example.  Can start w/ those and add more examples if needed (because they aint perfect).

Reply:
I am not sure I get this. This was there before which I didn't modify. Will discuss with you more.